airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3751,6 +3753,13 @@ components:
         pid:
           type: integer
           nullable: true
+        executor:
+          type: string
+          nullable: true
+          description: |
+            Executor the task is configured to run on or None (which indicates the default executor)
+
+            *New in version 2.10.0*
```

Comment:
Should this be 2.9.3?

Reply:
I prefer 2.10.

Reply:
Okay, I think I agree. But that does mean we can't merge this for a while, which is always unfortunate.

Reply:
I think we can? 2.9 (same as other minor versions) is released from a branch, not main.

Reply:
Correct. Patch releases are just cherry-picks on top of the minor branch (here 2.9)

Reply:
Yes. We refrained from merging things that had a lot of changes across the codebase because they made it difficult to cherry-pick other changes (so large refactorings basically). 

This one is pretty localised, so there is no reason to defer merging.

Generally speaking by default everything we merge now in main is targetted to 2.10 - and we only cherry-pick those things that we specifically think are good bug-fixes to cherry-pick to 2.9.1.

Also see: https://github.com/apache/airflow?tab=readme-ov-file#what-goes-into-the-next-release where this is described :).